tests/bluetooth/mesh/brg/prj.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_ZTEST=y
+CONFIG_ZTEST_MOCKING=y
```

Comment:
ZTEST mocking was deprecated. https://docs.zephyrproject.org/latest/develop/test/ztest_deprecated.html#
I'm not sure if it should be used for new tests.